action.yml

```diff
@@ -31,6 +31,10 @@ inputs:
   ALKILN_TAG_EXPRESSION:
     description: "The tag expression to use to limit which tests to run. Default is no tag expression."
     required: false
+  DOCASSEMBLECLI_VERSION:
+    description: 'Where to get the docassemblecli package. E.g. a GitHub zip or a pypi version.'
```

Comment:
Fairly certain this syntax `pip install docassemblecli==$DOCASSEMBLCLI_VERSION` will only work with specific pypi versions. You'd to wrap the whole thing in quotes for it to work with `@ https://github.com/.../archive/main.zip`.  https://pip.pypa.io/en/stable/reference/requirement-specifiers/ should have the different things `pip` accepts.

To be clear, I think restricting to pypi versions of the package is fine, but I just think that should be reflected in the description